judo/app/simulation.py

```diff
@@ -62,9 +62,9 @@ def step(self) -> None:
         """Step the simulation forward by one timestep."""
         if self.control is not None and not self.paused:
             try:
-                self.task.data.ctrl[:] = self.control(self.task.data.time)
+                self.sim_controls = self.control(self.task.data.time)
                 self.task.pre_sim_step()
-                mj_step(self.task.sim_model, self.task.data)
+                self.sim_backend.sim(self.task.sim_model, self.task.data, self.sim_controls)
```

Comment:
nit: i'd prefer to call this function `step()` to match our expectations from replacing `mj_step()`